Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/direct/messagemonitoring/impl/MessageMonitoringAPI.java

```diff
@@ -316,21 +316,15 @@ public void clearCache() {
 
     public boolean updateMessageMonitoringRetryCount(final MimeMessage message) {
 
-        try {
-            // get the message id
-            final String messageId = message.getMessageID();
+        // its already there, just update the
```

Comment:
Whoever originally wrote this comment has left me in suspense -- just update the what?? :smiley: